### DISCUSSION TITLE
IKEA LED1624G9 and power_on_behavior

### DISCUSSION THREAD

**Discussion by marthubner** [Koenkk/zigbee2mqtt#8456] [Devices]
Hi all,
I have multiple IKEA LED1624G9 bulbs and it seems this is the only IKEA bulb without "power_on_behavior" functionality which bothers me quite a bit.
Why is this so? Is it possible to add this functionality? Or would IKEA have to release new firmware update or something?

**Comment by sjorge** [comment]
Try changing https://github.com/Koenkk/zigbee-herdsman-converters/blob/2590aed44c2c6020fbc0e8a48b1e4227ad7af7b4/devices/ikea.js#L176 to

```
extend: tradfriExtend.light_onoff_brightness_colortemp_color(),
```

And see if it works

**Reply by marthubner** [reply]
Thank you very much for your suggestion. The "power_on_behavior" actually showed up in the exposes menu. But the bulb always goes on after turning on the power no matter what I set, so it just doesn´t work.
It is probably really the only IKEA bulb which doesn´t have that funcionality.

**Reply by sjorge** [reply]
Yeah seems likely it’s just that bulb then.
I know there WS G10 bulbs are also missing a bunch of features that the light panels and E14/E27 WS bulbs are missing :(
